src/Voron/Data/Tables/Table.cs

```diff
@@ -524,7 +524,7 @@ internal Tree GetTree(Slice name)
         private Tree GetTree(TableSchema.SchemaIndexDef idx)
         {
             if (idx.IsGlobal)
-                return _tx.ReadTree(idx.Name);
+                return _tx.ReadTree(idx.NameAsSlice.ToString());
```

Comment:
I created a new bug on this issue, as this is something that is happening throughout the entire code base and should take considerable, long-term, effort. See: http://issues.hibernatingrhinos.com/issue/RavenDB-5291 .